joint_teapot/workers/gitea.py

```diff
@@ -1,3 +1,4 @@
+<<<<<<< HEAD
```

Comment:
remove

Reply:
I did not push this... Appears that you forgot to remove it in a merge. Anyway I fixed it for you.

Reply:
I did not push this... Appears that you forgot to remove it in a merge. Anyway I fixed it for you.

Reply:
https://github.com/RealSaltyFish/Joint-Teapot/commit/667b9cd53baaeab39292687f90dc400b288437d1

Reply:
resolved